### PR TITLE
Add support for HTMLUnknownElement

### LINKED ISSUE #28: Add support for HTMLUnknownElement
The global scope is currently not populated with the `HTMLUnknownElement` interface.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLUnknownElement

It would be great to have this feature.

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
Fixes #28

### CODE REVIEW

File: basichtml.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const utils = require('./src/utils');
 const CustomElementRegistry = require('./src/CustomElementRegistry');
 const Document = require('./src/Document');
 const HTMLElement = require('./src/HTMLElement');
+const HTMLUnknownElement = require('./src/HTMLElement');
```
Comment (WebReflection) [inline basichtml.js]: this makes no sense. It should be `class extends HTMLElement {}`, not exactly the same class. Yet I don't see any valid use case for that in here.

Comment (SheepFromHeaven) [reply basichtml.js]: @WebReflection fixed the import you were right. What do you mean, you don't see a use case in here?

Comment (WebReflection) [reply basichtml.js]: I mean: what are you trying to solve ? why do you need `HTMLUnknownElement` ? what is the use case ?

Comment (SheepFromHeaven) [reply basichtml.js]: See #28 for discussion 👍

**Pull request merged**